src/NUnitFramework/tests/Attributes/PairwiseTests.cs

```diff
@@ -38,13 +38,13 @@ public class LiveTest
             private PairCounter pairsTested = new PairCounter();
 
             [OneTimeSetUp]
-            public void TestFixtureSetUp()
+            public void OneTimeSetUp()
```

Comment:
Unneeded but OK. We often refer to the OneTimeSetUp of a fixture this way in the docs.